tasks/request_files/request_files.py

```diff
@@ -12,55 +12,61 @@
 
 # noinspection PyPackageRequirements
 import boto3
+
 # noinspection PyPackageRequirements
 from botocore.client import BaseClient
+
 # noinspection PyPackageRequirements
 from botocore.exceptions import ClientError
 from cumulus_logger import CumulusLogger
 from run_cumulus_task import run_cumulus_task
 
+# from orca_shared import shared_recovery
+import shared_recovery
 
-# todo: Move to shared lib after ORCA-170
-class RequestMethod(Enum):
-    POST = 'post'
-    PUT = 'put'
+# # todo: Move to shared lib after ORCA-170
+# class RequestMethod(Enum):
+#     POST = 'post'
+#     PUT = 'put'
 
 
 DEFAULT_RESTORE_EXPIRE_DAYS = 5
 DEFAULT_MAX_REQUEST_RETRIES = 2
 DEFAULT_RESTORE_RETRY_SLEEP_SECS = 0
-DEFAULT_RESTORE_RETRIEVAL_TYPE = 'Standard'
+DEFAULT_RESTORE_RETRIEVAL_TYPE = "Standard"
```

Comment:
Any particular reason you're switching from ' to "?

Reply:
Makes the PR a bit noisy is all.

Reply:
I agree